src/components/WeatherShowcase/WeatherShowcase.tsx

```diff
@@ -1,10 +1,11 @@
+/* eslint-disable no-console */
```

Comment:
Do you really want to disable the eslint warning for the whole file? What if a developer leaves a `console.log` during testing and forgets to remove it, we want the linter to flag this. It's much better to disable a single line for this reason.